### PR TITLE
Intake Motor Changes

### PR DESCRIPTION
**Pull request by kirbt**
Intake only has one motor now so I removed some code accordingly.

### CODE REVIEW

**Review by TAKBS2412 — APPROVED** [review]

**Review by bobtseattle — APPROVED** [review]
Approved

File: src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java [hunk 1]
```diff
@@ -96,45 +83,42 @@ public IntakeSubsystem(WPI_TalonFX motorOuterAxle,
...
     }
 
     /**
-     * Stops motors and updates motor state
+     * Stops motor and updates motor state
      */
     public void intakeStop() {
-        motorOuterAxle.stopMotor();
-        motorInnerAxle.stopMotor();
+        motor.set(0);
```
Comment (SumedhP) [inline src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java:120]: Is there a reason you are both stopping the motor and then also telling it to coast to a stop after

File: src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java [hunk 2]
```diff
@@ -96,45 +83,42 @@ public IntakeSubsystem(WPI_TalonFX motorOuterAxle,
...
         }
     }
 
     /**
-     * Spins motors outwards and updates motor state.
+     * Spins motor outwards and updates motor state.
      * Runs if Intake is extended which is when the solenoid is retracted.
      */
     public void intakeOut() {
         if (intakeSolenoidState == RETRACT) {
```
Comment (SumedhP) [inline src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java:110]: This is the same as isIntakeExtended method, can you implement that instead for easier readability

File: src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java (hunk 1 above)
Comment (kirbt) [reply src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java:120]: oops i forgot to remove stopMotor();

**Review by SumedhP — APPROVED** [review]

**Pull request merged**